contracts/control/unstructured/UOwnable.sol

```diff
@@ -48,9 +48,7 @@ abstract contract UOwnable is UInitializable {
      * @notice Accepts and transfers the ownership of the contract to the pending owner
      * @dev Can only be called by the pending owner to ensure correctness
      */
-    function acceptOwner() external {
-        if (msg.sender != pendingOwner()) revert UOwnableNotPendingOwnerError(msg.sender);
-
+    function acceptOwner() external onlyPendingOwner {
```

Comment:
I think we should put back pending owner logic also since there isn't a good reason to surface `onlyPendingOwner`.